lib/logproto/tests/test-regexp-multiline-server.c

```diff
@@ -212,12 +252,5 @@ test_first_line_without_prefix(LogTransportMockConstructor log_transport_mock_ne
   assert_proto_server_fetch(proto, "Foo Multiline\nmulti", -1);
 
   log_proto_server_free(proto);
-  log_proto_server_options_destroy(&proto_server_options);
```

Comment:
For both reviewing and later (for example bisect) it is better to split up changes that are independent.
The fix of double free and change in the test could have been a separate commit.

Reply:
Thanks a lot for the comment. I will do it this way in the future for cases like this. Really good to know those best practices. Appreciated!